### PR TITLE
Fix ocasional panic in UTF16ToStringArray when processing unicode strings

### PR DESCRIPTION
**Pull request by narph**
Should also fix https://github.com/elastic/beats/issues/15264.
Reproducing steps based on the original issue:
- system language set to german
- config for perfmon should contain:
```
    - instance_label: physical_disk.name
      measurement_label: physical_disk.write.time.pct
      query: '\PhysicalDisk(*)\% Disk Write Time'
```
the UTF16ToStringArray will not be able to read the next line start.

### CODE REVIEW

File: metricbeat/module/windows/perfmon/pdh_query_windows.go [hunk 1]
```diff
@@ -274,7 +274,7 @@ func UTF16ToStringArray(buf []uint16) []string {
 	stringLine := UTF16PtrToString(&buf[0])
 	for stringLine != "" {
 		strings = append(strings, stringLine)
-		nextLineStart += len(stringLine) + 1
+		nextLineStart += len([]rune(stringLine)) + 1
```
Comment (kaiyan-sheng) [inline metricbeat/module/windows/perfmon/pdh_query_windows.go]: Is it possible to add unit tests for this function/or this specific case?

Comment (narph) [reply metricbeat/module/windows/perfmon/pdh_query_windows.go]: I have added a test for this

File: metricbeat/module/windows/perfmon/pdh_query_windows_test.go [hunk 2]
```diff
@@ -114,3 +115,19 @@ func TestObjectNameRegexp(t *testing.T) {
 		}
 	}
 }
+
+func TestUTF16ToStringArray(t *testing.T) {
+	var array = []string{"\\\\DESKTOP\\Fysieke schijf(0 C:)\\Huidige wachtrijlengte op de schijf", "\\\\DESKTOP\\Fysieke schijf(_Total)\\Percentage schijftijd", ""}
```
Comment (jsoriano) [inline metricbeat/module/windows/perfmon/pdh_query_windows_test.go]: Do these strings reproduce this problem? Should we introduce some non-ascii character?

Comment (narph) [reply metricbeat/module/windows/perfmon/pdh_query_windows_test.go]: I have updated the test samples with the problematic values

Comment (jsoriano) [reply metricbeat/module/windows/perfmon/pdh_query_windows_test.go]: Thanks!

**Pull request merged**